components/clinical/eForm/src/index.tsx

```diff
@@ -44,9 +46,15 @@ const EForm: FC<Props> = ({ url, callback, ...rest }) => {
     }
   }, [callback])
 
+  useLayoutEffect(() => {
+    if (forceRefresh && iframeRef.current) {
+      setIframeKey((prevKey: number) => prevKey + 1)
+    }
+  }, [forceRefresh])
+
   return (
     <StyledIframe {...rest}>
-      <iframe src={url} title="eForm" />
+      <iframe key={iframeKey} ref={iframeRef} src={url} title="eForm" data-iframe-key={iframeKey} />
```

Comment:
Can we remove key now and just use data-iframe-key?

Reply:
Oh yeah, good spot